backends/backend-sdl/src/arc/backend/sdl/SdlApplication.java

```diff
@@ -2,12 +2,15 @@
 
 import arc.*;
 import arc.audio.*;
+import arc.files.*;
 import arc.func.*;
 import arc.graphics.*;
 import arc.struct.*;
 import arc.util.*;
 import arc.util.async.*;
 
+import java.io.*;
+import java.lang.management.*;
```

Comment:
As I mentioned, you can't import this directly, it'll throw an exception and crash instantly, because I only include the `java.base` module on my packaged JDK. You can only access it with reflection and `Class.forName` with a try/catch.

Reply:
Oh god

Reply:
![image](https://user-images.githubusercontent.com/26940575/144686826-98358e25-57d1-457e-b1d8-e11fe51e95ed.png)
This refuses to work no matter how hard i try

Reply:
I think im just suffering from smooth brain again

Reply:
You're invoking the method with `null` like it's static, but it's not, it takes the bean instance.

Reply:
Yeah, i figured that one out lol, anyways now i get this `class arc.backend.sdl.SdlApplication cannot access class sun.management.RuntimeImpl (in module java.management) because module java.management does not export sun.management to unnamed module @5b87ed94`

Reply:
Don't use `bean.getClass()`, use `Class.forName("fqcn of RuntimeMXBean")`

Reply:
Ah ok